extensions/wikia/PortableInfoboxBuilder/services/PortableInfoboxBuilderService.class.php

```diff
@@ -245,12 +246,21 @@ protected function importNodeToDom( $domDocument, $childNodeDom ) {
 	}
 
 	/**
+	 * @param $collapse
 	 * @return DOMElement
 	 */
-	protected function createGroupDom() {
-		return dom_import_simplexml(
-			new SimpleXMLElement( '<' . \Wikia\PortableInfoboxBuilder\Nodes\NodeGroup::XML_TAG_NAME . '/>' )
-		);
+	protected function createGroupDom($childHeader, $childNodeDom) {
+		$groupElem = new SimpleXMLElement( '<' . \Wikia\PortableInfoboxBuilder\Nodes\NodeGroup::XML_TAG_NAME . '/>' );
+		if ( $childHeader['section_collapsible'] ) {
+			$groupElem->addAttribute('collapse', 'open');
+		}
+
+		// clean the mess connected to differences in xml tree and data model on mercury side
+		$childNodeDom->removeAttribute('section_collapsible');
```

Comment:
Please create a new header instead or modify it in the generic flow of getFormattedMarkup in which it is created, as we discussed offline.